### DISCUSSION TITLE
Global Settings for used/available Embeds

### DISCUSSION THREAD

**Discussion by PhilReinking** [outline/outline#1926] [Ideas]
Just saw issue #1923, which is about the growing amount of available embeds. I wondered if there is a plan to introduce a settings page, where a admin user can change the embeds that should be available in the editor.

It could be as simple as a list of all embeds with a checkbox toggle to activate and deactivate an embed service.

**Comment by tommoor** [comment]
There hasn't been a plan for this but it seems reasonable, are there embeds that you would turn off if this functionality existed?
